### DISCUSSION TITLE
Galaxy Badge

### DISCUSSION THREAD

**Discussion by jafludo** [community/community#18736] [Other Feature Feedback, Questions, & Ideas]
Hello,
How to get the badge dudes???

**Comment by notyashhh** [comment]
Galaxy Badge             -> You can answer any two discussion questions.
Pull Shark                   -> Get any 2 PR's approved & merged into a branch.
YOLO (kinda glitchy) -> Merge your PR without an approval.

**Reply by oxSTERBENxo** [reply]
I've heard it doesn't work in the community though, they've removed it bc of cheating? You can try in discussions in other rep.

**Reply by oxSTERBENxo** [reply]
❤️

**Comment by Ty-Irving** [comment]
https://github.com/Schweinepriester/github-profile-achievements

**Comment by ebndev** [comment]
Hi @jafludo, thanks for being a part of the GitHub Community!

We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).
